.circleci/config.yml

```diff
@@ -65,6 +65,15 @@ jobs:
       - <<: *restore_dependency_cache
       - run: npm run test:examples
 
+  # test_macos:
```

Comment:
I probably missed a thread/conversation somewhere. Why is this commented out?

Reply:
To save it into the git history for later use. It can be removed any time after this is merged.

Reply:
Please don't do that. We don't need to version commented out code.

If you're worried about this stuff getting "lost", just create a branch with the changes in it, or host as `.patch` file as a Gist for future use.